test failure
```
======================================================================
ERROR: corehq.apps.hqadmin.tests.test_views:AdminRestoreViewTests.test_get_context_data
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/hqadmin/tests/test_views.py", line 23, in test_get_context_data
    context = view.get_context_data(foo='bar', view='AdminRestoreView')
  File "/mnt/commcare-hq-ro/corehq/apps/hqadmin/views.py", line 533, in get_context_data
    'num_locations': num_locations,
UnboundLocalError: local variable 'num_locations' referenced before assignment
----------------------------------------------------------------------
```

--- a/corehq/apps/hqadmin/tests/test_views.py
+++ b/corehq/apps/hqadmin/tests/test_views.py
@@ -30,4 +30,5 @@ class AdminRestoreViewTests(SimpleTestCase):
                 'status_code': 500,
                 'timing_data': [],
                 'num_cases': 0,
+                'num_locations': 0,
             })

--- a/corehq/apps/hqadmin/views.py
+++ b/corehq/apps/hqadmin/views.py
@@ -523,6 +523,7 @@ class AdminRestoreView(TemplateView):
                 xml_payload = E.error(message)
             restore_id_element = None
             num_cases = 0
+            num_locations = 0
         formatted_payload = etree.tostring(xml_payload, pretty_print=True)
         context.update({
             'payload': formatted_payload,

